### ISSUE TITLE
Tabs rendered via Livewire do not have first tab selected by default

### ISSUE THREAD

**Issue by gaelicwinter** [livewire/flux#867]
### Flux version

v1.0.32

### Livewire version

v3.5.16

### What is the problem?

I have a component that I mount and render on a button click, so that its contents are determined after the click and Livewire handles the html swap.  If the content includes flux tabs, then even though the first tab appears selected, its corresponding panel remains hidden until the tab button is clicked by the user.  There are other components on the page that also include flux tab groups but since they are rendered on page load, they do not suffer the same problem.

Here is a simple demo I have made to show the issue. Or perhaps I am doing something wrong?

### Code snippets

```
<?php

use Livewire\Volt\Component;

new class extends Component {

    public $tabs = [];
    public $activeTab = 'tab-1';

    public function getTabs() {
        $this->tabs = [
            ['title' => 'Tab 1', 'type' => 'tab-1', 'content' => 'Tab 1 Content'],
            ['title' => 'Tab 2', 'type' => 'tab-2', 'content' => 'Tab 2 Content'],
            ['title' => 'Tab 3', 'type' => 'tab-3', 'content' => 'Tab 3 Content'],
        ];
    }

    public function showContent()
    {
        $this->getTabs();
    }
}; ?>

<div>
    <flux:button wire:click="showContent">Show Content</flux:button>

    <flux:tab.group>
        @foreach($tabs as $tabId => $tab)
            <flux:tab.panel name="{{ $tab['type'] }}" wire:key="panel-{{ $tab['type'] }}">
                {{ $tab['content'] }}
            </flux:tab.panel>
        @endforeach
        <flux:tabs wire:model="activeTab">
            @foreach($tabs as $tabId => $tab)
                <flux:tab name="{{ $tab['type'] }}" wire:key="{{ $tab['type'] }}">
                    {{ $tab['title'] }}
                </flux:tab>
            @endforeach
        </flux:tabs>
    </flux:tab.group>
</div>

```
You can use this line on any page to invoke the component:

```
@livewire('tab-test')
```

### How do you expect it to work?

When the showContent method is called and the tab group is rendered, the tab indicated by the activeTab property (or a default) should automatically be selected and its corresponding panel should be visible.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by jeffchown** [comment]
@gaelicwinter I tried your code and can reproduce the issue.  While it looks like the correct tab is selected, the selected tab's panel is not displayed (as you say, until that tab is clicked).

I believe this is related to issue: https://github.com/livewire/flux/issues/646
@calebporzio explained the specifics in his comment in that issue: https://github.com/livewire/flux/issues/646

@calebporzio , the tabs in this issue that are added dynamically when the 'Show Content' button is clicked are rendering correctly (thanks to your fix for issue #646) - however, the current/selected tab's panel is **not** assigned the `data-selected` attribute, nor is its content being displayed.

I went back and re-tested issue #646 and found the same issue - new tabs dynamically added are displayed correctly, but their panel content is not displayed (if that tab is programmatically made current/active upon its addition).

**Comment by gaelicwinter** [comment]
Thanks for this, I'm not crazy :-).  When I first noticed the issue I did go and implement [https://github.com/livewire/flux/issues/646]#646 to see what would happen and noticed the same problem.  Another thing I found is that I don't seem to be able to trigger a click event on the tab button?  Maybe that's just me or maybe its related to this, since triggering a click is just a FE method of changing the livewire property in the component.
